apps/idos-data-dashboard/prisma/migrations/20260619114242_change_index_to_unique_combination/migration.sql

```
@@ -0,0 +1,11 @@
+/*
+  Warnings:
+
+  - A unique constraint covering the columns `[wallet_address,wallet_type,wallet_public_key]` on the table `User` will be added. If there are existing duplicate values, this will fail.
+
+*/
+-- DropIndex
+DROP INDEX "User_wallet_address_wallet_type_key";
```

SQLFluff (4.2.2)

[error] DROP statement should use CONCURRENTLY to avoid locking the table.

(PG01)